flow/flow.go

```diff
@@ -894,6 +899,123 @@ func (f *Flow) newTransportLayer(packet *Packet, opts Opts) error {
 
 	return nil
 }
+func newDNSRecordFromLayersRecord(rr *layers.DNSResourceRecord) fl.DNSResourceRecord {
```

Comment:
can you move this function into flow/layers/dns.go, please see my comment below about splitting protocols

Reply:
done. Also moved all other DNS-related functions